### PR TITLE
Added multiple database Active Record guide entry to the release notes

### PR DESCRIPTION
**Pull request by abhaynikam**
ref: https://github.com/rails/rails/pull/36820#issuecomment-516835647

Added the entry in the release notes for new guide.

### CODE REVIEW

**Comment by bogdanvlviv** [comment]
This should go to the `master` first and then we backport to the `6-0-stable`.

**Comment by abhaynikam** [comment]
Oh. I didn't knew that. Let me fix it 😊

**Comment by abhaynikam** [comment]
I am sorry about the labels, it happened because I changed the base to `6-0-stable` to `master`. This is `docs` only change.

File: guides/source/6_0_release_notes.md [hunk 1]
```diff
@@ -1001,6 +1001,8 @@ Ruby on Rails Guides
 Please refer to the [Changelog][guides] for detailed changes.
 
 ### Notable changes
+*   Add Multiple Databases with Active Record guide.
```
Comment (bogdanvlviv) [inline guides/source/6_0_release_notes.md]: Can we add an empty line between "Notable changes" and this line for consistency with other sections?

**Review by bogdanvlviv — APPROVED** [review]
Thank you!

**Pull request merged**